### DISCUSSION TITLE
An AI code explainer would definitely speed up learning progress for LuaSnip's beginners.

### DISCUSSION THREAD

**Discussion by no-vici** [L3MON4D3/LuaSnip#905] [Ideas]
Hi beginners, 

If you are beginners like me, you should have a look at [ZZZ Code.AI](https://zzzcode.ai/answer-question?id=20eb3fef-ac74-4d96-8019-23c7ebc2a499), and I think DOC writers should use AI to alleviate some of the tedious stages too ;)

**Comment by L3MON4D3** [comment]
Oh wow, it does get a lot right (I think all of it except for some ambiguous wording)
Thank you for sharing :+1:

**Reply by no-vici** [reply]
> Oh wow, it does get a lot right (I think all of it except for some ambiguous wording) Thank you for sharing +1

ZZZ is still in its infantry, I'm reading the DOC again and found some concepts a bit difficult to grasp (I don't blame the DOC writer, I'm so dumb), and here it comes, ZZZ has helped clear the clouds. 

Very grateful to those behind the project, they are great and generous like you.
